corehq/apps/app_manager/templates/app_manager/case_summary.html

```diff
@@ -135,30 +137,47 @@ <h4 class="panel-title panel-title-nolink">
                                         </td>
                                         <!-- ko if: !$index() -->
                                         <td>
-                                            <ul class="list-unstyled" data-bind="foreach: $parent.short_details">
-                                                <li>
-                                                    <span data-bind="html: $root.moduleReference($data.module_id)"></span> "<!-- ko text: $root.translate($data.header) --><!-- /ko -->"
-                                                </li>
-                                            </ul>
+                                            {% include 'app_manager/partials/case_summary_case_details.html' with detail='$parent.short_details' %}
                                         </td>
                                         <td>
-                                            <ul class="list-unstyled" data-bind="foreach: $parent.long_details">
-                                                <li>
-                                                    <span data-bind="html: $root.moduleReference($data.module_id)"></span> "<!-- ko text: $root.translate($data.header) --><!-- /ko -->"
-                                                </li>
-                                            </ul>
+                                            {% include 'app_manager/partials/case_summary_case_details.html' with detail='$parent.long_details' %}
                                         </td>
                                         <!-- /ko -->
                                     </tr>
                                 <!-- /ko -->
+                                <!-- ko if: !forms.length && (short_details.length || long_details.length) -->
```

Comment:
I just combed back through the code to verify that these are indeed plain arrays and not observable arrays, because one of the most common mistakes I make in knockout is thinking something is an observable when it isn't, or vice versa. Sometimes I think about having a standard where *all* knockout model properties are observables...but that gets to be kind of a pain when dealing with deeply nested objects (like this one). Thoughts?

Reply:
Yeah, I don't think having everything be an observable is desirable, especially with lists. It doesn't make sense for these to be observable since the data in them doesn't change, so they don't technically need to be "observed".